src/ia-book-actions.js

```diff
@@ -374,9 +383,11 @@ export default class IABookActions extends LitElement {
    * @param {Event} event
    */
   changeModalState(event) {
+    // this doesn't really work? shows empty button
     if (event.target === undefined) return;
 
     const button = event.target;
+    button.disabled = true;
     button.innerHTML = `<ia-activity-indicator
```

Comment:
issue: 
```
> keep reading -> sometimes hangs - modal stuck
```

when warning modal is shown && user clicks on Renew -> button is blank. upon inspection <ia-activity-indicator> is not loaded, but weirdly i added it to PB entry point and it sill didn't show up?
Q: can we use the timer intersitial from modalconfig `showProcessingIndicator`?